src/com/mincom/ellipse/javatraining/springmvc/controller/ApplicationController.java

```diff
@@ -16,6 +16,7 @@
  
 	private static final String INDEX_PAGE_KEY = "index";
 	private static final String FINAL_PAGE_KEY = "finalPage";
+    private static final String ERROR_PAGE_KEY = "errorPage";
 	private final String MESSAGE_MAP_KEY = "input_message";
 	private final static Logger logger = LoggerFactory.getLogger(ApplicationController.class);
```

Comment:
`private static final` is preferred, the constant name should be all capitalized